README.md

```diff
@@ -1,37 +1,45 @@
-# TODO-MVP-Loaders
+# todo-mvp-loaders
 
-It is based on the [TODO-MVP](https://github.com/googlesamples/android-architecture/tree/todo-mvp/todoapp) sample and uses Loaders to get the data from the tasks repository.
+This version of the app is called todo-mvp-loaders, and uses the [Loaders API](https://developer.android.com/guide/components/loaders.html) to load data from the tasks repository. The Loaders API provides the following advantages to this sample:
 
-<img src="https://github.com/googlesamples/android-architecture/wiki/images/mvp-loaders.png" alt="Diagram"/>
+ * Loaders load data asynchronously, helping to free up the main thread and improving UI responsiveness.
+ * Loaders are used to implement an observer which monitors the repository data source for changes, and refreshes results in the UI.
+ * Loaders automatically persist query results when changes occur in configuration, such as when a device is rotated.
```

Comment:
Does "persist query results" come from other documentation sources? This is changing the meaning considerably from " automatically reconnect to the last loader".